examples/job_script.sbatch

```diff
@@ -15,12 +15,19 @@
 #SBATCH --mem=0
 #SBATCH --output="%x-%j.log"
 
-pushd ..
-source setup.sh
-popd
+JOB_SCRIPT=$(scontrol show job ${SLURM_JOB_ID} | awk -F= '/Command=/{print $2}')
+JOB_SCRIPT_DIR=$(dirname ${JOB_SCRIPT})
```

Comment:
Yesterday I also tried something similar to what you proposed but unfortunately it doesn't work. It is a bit more complicated because SLURM copies the script somewhere else in the file system.  Running a sbatch script with this:
```
JOB_SCRIPT=$(scontrol show job ${SLURM_JOB_ID} | awk -F= '/Command=/{print $2}')
echo ${JOB_SCRIPT}
echo ${BASH_SOURCE[0]} 
```
produces:
```
/gpfs/bbp.cscs.ch/data/scratch/proj16/cantarut/AstroVascPy/examples/test_script.sbatch
/var/slurmd/job2498085/slurm_script
```

Reply:
I still have problems, even with the latest approach.  I'm working on it.